cmd/server/main.go

```diff
@@ -26,7 +26,7 @@ func main() {
 	oplog.SetupDefaults()
 
 	app := cli.NewApp()
-	app.Flags = cliapp.ProtectFlags(flags.Flags)
+	app.Flags = cliapp.ProtectFlags(config.Flags)
```

Comment:
why this change? I typicall prefer to keep config and flags in separate packages

Reply:
I'm not 100% why this was moved (I believe it was part of the changeset when I picked up this task). But my guess is that this was to resolve circular imports, which got nasty when I tried a bit of reorganizing.

Reply:
See [this comment](https://github.com/Layr-Labs/eigenda-proxy/pull/321#discussion_r1979874780)